File: Examples/VisualBasic/Images/GetInformationOfImages.vb
Imports System.IO
Imports AsposeNote = Aspose.Note
Imports System.Collections.Generic

Namespace Images
    Public Class GetInformationOfImages
        Public Shared Sub Run()
            ' ExStart:GetInformationOfImages
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Images()

            ' Load the document into Aspose.Note.
            Dim oneFile As New Document(dataDir & Convert.ToString("Aspose.one"))

            ' Get all Image nodes
            Dim images As IList(Of AsposeNote.Image) = oneFile.GetChildNodes(Of AsposeNote.Image)()

            For Each image As AsposeNote.Image In images
                Console.WriteLine("Width: {0}", image.Width)
                Console.WriteLine("Height: {0}", image.Height)
                Console.WriteLine("OriginalWidth: {0}", image.OriginalWidth)
                Console.WriteLine("OriginalHeight: {0}", image.OriginalHeight)
                Console.WriteLine("FileName: {0}", image.FileName)
                Console.WriteLine("Extension: {0}", image.Extension)
                Console.WriteLine("LastModifiedTime: {0}", image.LastModifiedTime)
                Console.WriteLine()
            Next
            ' ExEnd:GetInformationOfImages
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Images/GetInformationOfImages.cs
using System.IO;
using Aspose.Note;
using System.Collections.Generic;
using System;

namespace Aspose.Note.Examples.CSharp.Images
{
    public class GetInformationOfImages
    {
        public static void Run()
        {
            // ExStart:GetInformationOfImages
            // ExFor:Document
            // ExFor:CompositeNode`1.GetChildNodes
            // ExFor:Image
            // ExFor:Image.Width
            // ExFor:Image.Height
            // ExFor:Image.OriginalWidth
            // ExFor:Image.OriginalHeight
            // ExFor:Image.FileName
            // ExFor:Image.LastModifiedTime
            // ExSummary:Shows how to get image's meta information.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Images();

            // Load the document into Aspose.Note.
            Document oneFile = new Document(dataDir + "Aspose.one");

            // Get all Image nodes
            IList<Aspose.Note.Image> images = oneFile.GetChildNodes<Aspose.Note.Image>();

            foreach (Aspose.Note.Image image in images)
            {
                Console.WriteLine("Width: {0}", image.Width);
                Console.WriteLine("Height: {0}", image.Height);
                Console.WriteLine("OriginalWidth: {0}", image.OriginalWidth);
                Console.WriteLine("OriginalHeight: {0}", image.OriginalHeight);
                Console.WriteLine("FileName: {0}", image.FileName);
                Console.WriteLine("LastModifiedTime: {0}", image.LastModifiedTime);
                Console.WriteLine();
            }

            // ExEnd:GetInformationOfImages
        }
    }
}